feat(scully): logrocket render plugin.

GuzmanPI opened the pull request:
The docs app uses this plugin.

Implements the pluguin

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/main/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other... Please describe:

LogRocket plugin for the docs. This may be an npm plugin in the future.

## What is the current behavior?

<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: #673 

## What is the new behavior?

Adds LogRocket's scripts to the <head> tag.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->

## Other information

GuzmanPI committed:
feat(scully): Add logrocket render plugin.

Fixes tests adds dummy app and id

SanderElias commented on libs/plugins/logrocket/src/lib/plugins-logrocket.ts:
```diff
@@ -0,0 +1,25 @@
+  html: string,
+  route: HandledRoute
+): Promise<string> => {
+  const logrocketConfig = getMyConfig('logrocket');
```
```suggestion
  const logrocketConfig = getMyConfig(logrocketPlugin);
```
Works better with just the plugin itself

GuzmanPI replied:
I noticed you used the plugin itself in the bas-href-rewrite, but I was not sure how to do it. Thx @SanderElias!

GuzmanPI replied:
I think more than a default config, it should throw an error if there is no config. I'll make the changes

SanderElias reviewed (approved):
Please change the thing I highlighted!
Also, shouldn't you define a default config in there?

BTW, you don't need to blush it to be used in scully.

jorgeucano commented:
@GuzmanPI I check the test and show this error

```    +     </style><script>window['ScullyIO']='generated';</script>
    +     <script src="https://cdn.logrocket.io/LogRocket.min.js"></script>
    +     <script>window.LogRocket && window.LogRocket.init('undefined/undefined');</script></head>``` 
Please check the config is not working!

GuzmanPI commented:
I'll check the tests

jorgeucano committed:
feat(Scully/publish/plugins): add support for logrocket

Add keys for logrocket
fix issue undefined values in getMyConfig
add prod option for support
into scully config
change web publisher for publish with logrocket

jorgeucano committed:
update test for logrocket

jorgeucano committed:
test(lib): add test for LogRocket

jorgeucano committed:
fix(docsWeb): fix typo UTF-8

jorgeucano committed:
fix(monorepo): remove jasmine dependencies

jorgeucano committed:
fix snapshot

jorgeucano committed:
fix(test-jest): update expect import

jorgeucano committed:
add tslib

jorgeucano committed:
change test for md

jorgeucano committed:
change test for md

Commit:
update test

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/scullyio/scully?pullRequest=680) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/scullyio/scully?pullRequest=680) before we can accept your contribution.<br/>**2** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: GuzmanPI<br/>:white_check_mark: jorgeucano<br/>:x: HeroDevs<br/><hr/>**HeroDevs** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/scullyio/scully?pullRequest=680) it.</sub>
